elytron/pom.xml

```diff
@@ -26,7 +26,7 @@
         <version>19.0.0.Final-SNAPSHOT</version>
     </parent>
 
-    <artifactId>wildfly-elytron-integration</artifactId>
+    <artifactId>wildfly-elytron-integration-legacy-namespace</artifactId>
 
     <name>WildFly: Elytron Subsystem</name>
```

Comment:
Minor, for the shake of completeness, this should be changed to "WildFly: Elytron Subsystem (non-Jakarta Namespace)"

Reply:
@bstansberry wasn't a fan of such change in wildfly when I pushed the first version of similar change so I didn't introduce it here either (even when I think it makes sense, it is just one more place to look for when doing the migration which is a good argument not to).

I can add it if you want it though.

Reply:
Ahh, ok, I thought we wanted them. Then, we are fine as it is, thanks